README.md

```diff
@@ -75,61 +69,89 @@ Please note that using Ollama requires a large (>4GB) download of the docker ima
     docker compose exec ollama ollama pull tinyllama
     ```
 
-### Create a table, run a vectorizer, and perform semantic search
-
-1. **Connect to the database in your local developer environment**
-   The easiest way connect to the database is with the following command:
-   `docker compose exec -it db psql`. 
-   
-   Alternatively, you can connect to the database with the following connection string: `postgres://postgres:postgres@localhost:5432/postgres`.
+### Create a FastAPI app that performs semantic search and RAG over wikipedia articles
 
-1. **Enable pgai on your database**
+You can take a look at a simple [fastAPI Application](/examples/simple_fastapi_app/main.py) to see how to
+setup an app to perform RAG with pgai Vectorizer. We'll walk you through the main parts of the code below. 
```

Comment:
I wonder if an integration with pydantic.ai would be more relevant to users. Not sure how much people build agents and how much they build more "traditional" applications with just a bit of AI seasoning.